pep-0723.rst

```diff
@@ -338,6 +344,19 @@ strings (``'''``) be used. For example:
     """
 
 
+Tooling buy-in
+==============
+
+The following is a list of tools that have expressed support for this PEP or
+have committed to implementing support should it be accepted:
+
+* `Pantsbuild <https://discuss.python.org/t/31151/15>`__
+* `Pex <https://discuss.python.org/t/31151/15>`__
```

Comment:
I think you should distinguish mypy/ruff (which have expressed support for the embedded config part of the PEP), vs hatch (which I suppose would support all of it) vs pex/pantsbuild (which have said they'll implement 722 or 723, but have endorsed neither).

Reply:
Done!